libs/drawer/src/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Flex, Heading, Slide, Stack } from '@chakra-ui/react';
+import React from 'react';
+
+import { IconClose } from '@millicast-react/dolbyio-icons';
+import IconButton from '@millicast-react/icon-button';
+
+import { DrawerProps } from './types';
+
+const Drawer = ({
+  children,
+  drawerProps: { background, bg, ...drawerProps } = {},
+  fullHeight,
+  heading,
+  headingProps = {},
+  isOpen,
+  onClose: handleDrawerClose,
+}: DrawerProps) => {
+  const drawerBackground = background ?? bg ?? 'dolbyNeutral.800';
+
+  return (
+    <Slide in={isOpen} style={{ position: 'absolute' }} unmountOnExit>
+      <Flex
```

Comment:
Is that possible we use Chakra UI's Text and Layer's styles to reduce the effort on text properties (font size, font weight, line height) or layer properties (bg, color, shadow), check this https://chakra-ui.com/docs/styled-system/text-and-layer-styles. I am thinking, we could save some effort in the future if we use that, because I already see there are a lot of duplicated properties in different components.

Reply:
I think that's a sensible idea moving forwards. Either that, or define styles and provide through the `sx` prop, we have a precedent for this as the pattern being [used in the `VideoView` component ](https://github.com/dolbyio-samples/rts-app-react-publisher-viewer/blob/4d567bd2ef64f9e885347a4c3c99f9a281ff0fab/libs/video-view/src/index.tsx#L72-L89).

Reply:
Sure, I leave this to you. If you want to do it in another branch, please create an issue then solve it later.

Reply:
Created issue https://github.com/orgs/dolbyio-samples/projects/1/views/1?pane=item&itemId=17338482